week-1/errors/3.js

```diff
@@ -1,6 +1,11 @@
 const cardNumber = 4533787178994213;
 const last4Digits = cardNumber.slice(-4);
 
+//the card number must have a string format if we want to use the slice method
+
+const cardNumber = "4533787178994213";
```

Comment:
Instead of adding a question mark to your number, you can convert it to a string. This will be more useful in the future. You can use the `toString()` method to convert the number to a string.